test/dynamo/test_misc.py

```diff
@@ -371,7 +370,6 @@ def unpack4(a, b):
             unpack4,
             2,
             expected_ops=5,
-            expected_ops_dynamic=ifdynstaticdefault(5, 7),
```

Comment:
Previous graph had
```
size = a.size()
getitem_2 = size[0]
getitem_3 = size[1];  size = None
```

size[0] is 5, and size[1] is s0, which is a graph placeholder, so both calls get removed (later in dynamo?)